Netflix/WebView.swift

```diff
@@ -0,0 +1,14 @@
+import Cocoa
+import WebKit
+
+class WebView: WKWebView {
+    var isInPipMode = false
```

Comment:
Not sure how, but this file is using spaces instead of tabs for indentation. It's not a deal breaker (it certainly doesn't affect the use of the app), but it'll stop SwiftLint from complaining at me when I build.

Reply:
Seems like Xcode project was set to use spaces (it is by default), I changed that option.